glide.lock

```diff
@@ -0,0 +1,12 @@
+hash: 791aaf9a24179856621317e12f08cf40a722596868353ecaa13f190a18daa7c3
```

Comment:
Could you add a small note on the development section of README explaining how to generate this automatically? (it it automatic, right?)